components/brave_wallet/common/solana_utils.cc

```diff
@@ -82,4 +85,27 @@ bool IsBase58EncodedSolanaPubkey(const std::string& key) {
   return Base58Decode(key, &bytes, kSolanaPubkeySize);
 }
 
+bool Uint8ArrayDecode(const std::string& str,
+                      std::vector<uint8_t>* ret,
+                      size_t len) {
+  if (!base::StartsWith(str, "[") && !base::EndsWith(str, "]"))
```

Comment:
As users usually copy-paste this string it makes sense to trim it before checking for []
Actually not sure if trimming already done somewhere by callers.

Reply:
Good point, trimming added in `KeyringService::ImportAccount`